README.Rmd

```diff
@@ -0,0 +1,114 @@
+---
+title: "Homework 5"
+author: "Ben Wang"
+date: "8/3/20"
+output: github_document
+---
+
+```{r setup, echo = FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(tidyverse)
+library(knitr)
+library(ggplot2)
+library(dplyr)
+library(readr)
+```
+
+## Part 1: Tidying up the dad_mom file
+
+```{r show P1 data, echo = FALSE}
+dad_mom <- read.delim("dad_mom.txt", header = TRUE)
+kable(dad_mom, format = "markdown")
+### Just taking a look at what we're working with
+```
+
+```{r tidying, echo = FALSE}
+tidy_dad_mom <- dad_mom %>%
+  unite(dad, name_dad, income_dad) %>%
+  unite(mom, name_mom, income_mom) %>%
+### Put together all the mom data under 'mom' and dad data under 'dad', now to put it together into a chart
+  gather(key = "Parent", value = "value", mom, dad) %>%
+  separate("value", into = c("Name", "Income"))
+### Print out the results
+kable(tidy_dad_mom, format = "markdown")
+```
+
+## Part 2: Joining together CTRP data
+
+```{r show P2 data, echo = FALSE}
+AUC_1 <- read.csv("CTRP_files/AUC_1.csv")
+AUC_2 <- read.csv("CTRP_files/AUC_2.csv")
+ccell_info <- read.csv("CTRP_files/cancer_cell_line_info.csv")
+cmpd_info <- read.csv("CTRP_files/compound_info.csv")
+expt_info <- read.csv("CTRP_files/Experiment_info.csv")
+```
+
+```{r join, echo = FALSE}
+CTRP_total <- AUC_1 %>%
+  bind_rows(AUC_2) %>%
+  inner_join(expt_info, by = c("experiment_id" = "expt_id")) %>%
+  inner_join(ccell_info, by = "master_ccl_id") %>%
+  inner_join(cmpd_info, by = "master_cpd_id") 
+
+### print(CTRP_total)
+### Nevermind, don't print this out, the document is far too long
```

Comment:
I like to use glimpse( ) instead of print.  if you just want to get an idea of what the data looks like / the variables this is a good option for a quick overview without printing the entire thing.